XamlStyler.Package/StylerPackage.cs

```diff
@@ -201,6 +209,32 @@ private void Execute(Document document)
             }
         }
 
+        private string GetConfigPathForItem(string path)
+        {
+            try
+            {
+                if (string.IsNullOrWhiteSpace(path))
+                {
+                    return null;
+                }
+
+                while ((path = Path.GetDirectoryName(path)) != null)
```

Comment:
Personally, I don't think that is ideal as its not that uncommon to have multiple solutions in the same repo (Cimbalino Toolkit is one, but I know of others examples), so one would need to copy the settings file to each one of the solution folders.
Having said that, I'm perfectly fine to make the change if that's the common agreement here! 😄 
The fallback to the local/network-set config file is already happening when a file isn't found, so we are covered on that case!